dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1150,7 +1150,7 @@ def get_student_response(script, level, lesson, student_number)
   test "should get progress for section with section script" do
     Unit.stubs(:should_cache?).returns true
 
-    assert_queries 6 do
```

Comment:
Is this from the new ability going through section instructors, or something else?

Reply:
I think it's because it previous would go through [this](https://github.com/code-dot-org/code-dot-org/pull/55053/files#diff-69482e4f9562cc1ae23a18d01fd28ec2675fa12de11c3509480db95f9ffdee9fR196) line and not get to the second query because it would be false on the first part. Now it goes through `s.instructors.include?(user)` every time.